src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -116,4 +122,16 @@ public String convertBBCode(String text){
       return temp;
 
   }
+
+  private float getSentimentScore(String text) throws IOException {
+      Document doc = Document.newBuilder()
```

Comment:
Similar comments to the other file, more contextual variable names are super helpful :)

Reply:
changed "doc" to "message"